langchain4j-milvus/src/main/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStore.java

```diff
@@ -295,6 +333,12 @@ public static class Builder {
         private String metadataFieldName;
         private String vectorFieldName;
 
+
+        public Builder milvusClient(MilvusServiceClient milvusClient) {
```

Comment:
could you please add an IT that creates/tests `MilvusEmbeddingStore` from `MilvusServiceClient`?

Reply:
Done!